AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1683,6 +1683,18 @@ - (void)recursivelyClearFetchedData
   [self clearFetchedData];
 }
 
+- (ASInterfaceState)interfaceState
+{
+  return _interfaceState;
+}
+
+- (void)setInterfaceState:(ASInterfaceState)interfaceState
+{
+  if (interfaceState != _interfaceState) {
```

Comment:
Yes, absolutely.  I may finish the initial internal implementation without this, but once proven to be the correct API, this will definitely be developer-facing and could conceivably even replace the fetch/clear methods

Sent from my iPhone

> On Nov 23, 2015, at 4:27 PM, Levi McCallum <notifications@github.com> wrote:
> 
> In AsyncDisplayKit/ASDisplayNode.mm:
> 
> > @@ -1683,6 +1683,18 @@ - (void)recursivelyClearFetchedData
> >    [self clearFetchedData];
> >  }
> >  
> > +- (ASInterfaceState)interfaceState
> > +{
> > +  return _interfaceState;
> > +}
> > +
> > +- (void)setInterfaceState:(ASInterfaceState)interfaceState
> > +{
> > +  if (interfaceState != _interfaceState) {
> Might be helpful to introduce a callback method: -[ASDisplayNode interfaceStateDidChange:(ASInterfaceState)previousState] for subclassing.
> 
> —
> Reply to this email directly or view it on GitHub.
>